### PR TITLE
PBM-348 Describe the pbm backup-delete command

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T10:55:42Z
+   2020-04-20T13:07:34Z
+   2020-04-20T13:13:20Z
```
Comment (akira-kurogane) [inline doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt:8]:
```suggestion
   2020-04-20T13:13:20Z
   2020-04-20T13:07:34Z
   2020-04-20T10:55:42Z
```
The order of backups has been changed by https://jira.percona.com/browse/PBM-438. The top of the list will be the most recent now.

File: doc/source/.res/code-block/bash/pbm-delete-backup-older-than-timestamp.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T20:55:42Z
+   2020-04-20T23:47:34Z
+   2020-04-20T23:53:20Z
+   2020-04-21T02:16:33Z
```
Comment (akira-kurogane) [inline doc/source/.res/code-block/bash/pbm-delete-backup-older-than-timestamp.txt:9]:
```suggestion
   2020-04-21T02:16:33Z
   2020-04-20T23:53:20Z
   2020-04-20T23:47:34Z
   2020-04-20T20:55:42Z
```

File: doc/source/.res/code-block/bash/pbm-delete-backup.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.. code-block:: bash
+
+  #Get the backup name
+  $ pbm list
+  Backup history:
+    2020-04-20T10:55:42Z
+    2020-04-20T13:07:34Z
+    2020-04-20T13:13:20Z
+    2020-04-20T13:45:59Z
```
Comment (akira-kurogane) [inline doc/source/.res/code-block/bash/pbm-delete-backup.txt:9]:
```suggestion
    2020-04-20T13:45:59Z
    2020-04-20T13:13:20Z
    2020-04-20T13:07:34Z
    2020-04-20T10:55:42Z
```

File: doc/source/running.rst [hunk 4]
```diff
@@ -167,11 +167,42 @@ restore.
...
 
+.. _pbm.backup.delete:
+
 Deleting backups
 --------------------------------------------------------------------------------
 
-Use the relevant native ``rm`` command for the storage. E.g. for S3-compatible
-object stores use ``aws s3 rm <file name>``. For a filesystem-type storage, use
-normal shell command ``rm``.
+Use the |pbm-delete-backup| command to delete a specified backup.
```
Comment (akira-kurogane) [inline doc/source/running.rst:175]:
```suggestion
Use the |pbm-delete-backup| command to delete a specified backup, or all backups older than a specified time.
```

File: doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T13:13:20Z
+   2020-04-20T13:07:34Z
+   2020-04-20T10:55:42Z
+ #Delete backups older than the specified one
+ $ pbm delete-backup -f --older-than 2020-04-20T13:13:20Z
```
Comment (dAdAbird) [inline doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt:10]: `pbm --older-than` doesn't delete backups in relation to another backup.`--older-than` works only with timestamps in one of two formats: `2020-04-20T13:13:20`(! without `Z` in the end) or `2020-04-20`.

File: doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T13:13:20Z
+   2020-04-20T13:07:34Z
+   2020-04-20T10:55:42Z
```
Comment (dAdAbird) [inline doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt:8]: In the sake of consistency: `pbm list` displays history in reverse order - newest backups in the bottom. And backup history is always sorted in that order

File: doc/source/running.rst [hunk 7]
```diff
@@ -167,11 +167,43 @@ restore.
...
+By default, the |pbm-delete-backup| command asks for your confirmation
+to proceed with the deletion. To bypass it, add the ``-f`` or
+``--force`` flag.
+
+.. code-block:: bash
+
+  $ pbm delete-backup --force 2020-04-20T13:45:59Z
+
+To delete the set of backups that precede the specified backup, pass the ``--older-than`` flag to the |pbm-delete-backup|
+command. The specified backup is not deleted in this case.
```
Comment (dAdAbird) [inline doc/source/running.rst:199]: the same as above, `--older-than` accepts only timestamps (no backup names)

File: doc/source/running.rst [hunk 8]
```diff
@@ -167,11 +167,43 @@ restore.
...
+
+.. include:: .res/code-block/bash/pbm-delete-backup-older-than-backup.txt
+
+To delete all backups that were created before the specified
+timestamp, enter the timestamp in the ISO 8601 format as an argument
```
Comment (dAdAbird) [inline doc/source/running.rst:204]: pbm doesn't accept any  ISO 8601 formatted date (it's too broad, see https://en.wikipedia.org/wiki/ISO_8601)
accepted only 2020-04-20T13:45:59 and 2020-04-20

File: doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt (hunk 1 above)
Comment (dAdAbird) [reply doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt]: Actually no. The most recent remains in the bottom. PBM-438 didn't change the sorting order but fixed output slice fo `pbm list --size`

**Pull request merged**